index.html

```diff
@@ -2,58 +2,64 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>This is my website!</title>
+  <title>Alex Thomas</title>
 
-  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
 </head>
+
 <body>
   <nav class="top-nav">
-    <ul class="foo-bar">
-      <li class="foo-bar__item foo-bar__item--brand">My Portfolio</li>
-      <li class="foo-bar__item">About</li>
-      <li class="foo-bar__item Day One</li>
-      <li class="foo-bar__item">Cool Stuff</li>
+    <ul class="nav-list">
+      <li class="nav-list__item nav-list__item--brand">My Portfolio</li>
+      <li class="nav-list__item">About</li>
+      <li class="nav-list__item">Day One</li>
+      <li class="nav-list__item">Cool Stuff</li>
     </ul>
   </nav>
-  <header class="l">
-    <div class="l__container">
-      <h2 class="l__title">This is my portfolio!</h2>
-      <hr class="y y--y">
-      <p class="l__text">There may be many like it, so I should replace this text to show that I am reading.</p>
+
+  <header class="hero">
+    <div class="hero__container">
+      <h2 class="hero__title">This is my portfolio!</h2>
+      <hr class="divider divider--hero">
+      <p class="hero__text">There may be many like it, so I should replace this text to show that I am reading.</p>
     </div>
   </header>
-  <section class="x-x">
-    <div class="z-x">
-      <h3 class="z-x">I'm a student at the Iron Yard</h3>
-      <hr class="y"
+
+  <section class="section-header">
+    <div class="bg-pop">
+      <h3 class="section-header">I'm a student at the Iron Yard</h3>
+      <hr class="divider">
       <p>I should write something about myself here and make it crafty.</p>
     </div>
   </section>
-  <section class="x-x">
-    <div class="z-x">
-      <h3 class="z-x">My First Day</h3>
-      <hr class="y">
+
+  <section class="section-container">
+    <div class="bg-standard">
```

Comment:
having the `bg-standard` class on the _section_, and the `section-container` class on the _div_, will extend the background color the full width of the screen.